moped-editor/src/views/projects/projectView/ProjectComponents.js

```diff
@@ -147,38 +170,22 @@ const ProjectComponents = () => {
    */
   const componentsAvailable = data && data.moped_proj_components.length > 0;
 
-  /**
-   * Update the full collection
-   */
-  useEffect(() => {
-    // If we have features and our collection is empty, then update
-    if (
-      selectedComp === 0 &&
-      projectFeatureRecords.length > 0 &&
-      featureFullCollection === null
-    ) {
-      setFeatureFullCollection(
-        createFeatureCollectionFromProjectFeatures(projectFeatureRecords)
-      );
-    }
-  }, [selectedComp, projectFeatureRecords, featureFullCollection]);
-
   // Return loading if not in progress
   if (loading) return <CircularProgress />;
 
   if (error) return <div>{JSON.stringify(error)}</div>;
 
   return (
     <ApolloErrorHandler errors={error}>
+      {componentEditMode && (
+        <ProjectComponentEdit
+          componentId={selectedComp}
+          handleCancelEdit={handleCancelEdit}
+          projectRefetchFeatures={refetch}
+          projectFeatureCollection={featureFullCollection}
+        />
+      )}
```

Comment:
Optimizing a little bit the way things are organized (for styling). Basically, I've moved the ProjectComponentEdit component outside of its parent so that it's not affected by padding.